recipe/setup_compiler.sh

```diff
@@ -14,34 +23,56 @@ if [[ ! -d  $SRC_DIR/cf-compilers ]]; then
         "gfortran_impl_${cross_target_platform}=${gcc_version}"
       )
     fi
+    if [[ "${cross_target_platform}" != "osx-"* ]]; then
+      extra_pkgs+=(
+        "binutils_impl_${cross_target_platform}=${binutils_version}"
+        "${cross_target_stdlib}_${cross_target_platform}=${cross_target_stdlib_version}"
+      )
+    else
+      extra_pkgs+=(
+        "clang"
+        "cctools_${cross_target_platform}"
+        "ld64_${cross_target_platform}"
+      )
+    fi
     # Remove conda-forge/label/sysroot-with-crypt when GCC < 14 is dropped
-    conda create -p $SRC_DIR/cf-compilers -c conda-forge/label/sysroot-with-crypt -c conda-forge --yes --quiet \
-      "binutils_impl_${build_platform}" \
+    conda create -p ${CF_PREFIX} -c conda-forge/label/gcc-experimental -c conda-forge/label/sysroot-with-crypt -c conda-forge --use-local --yes --quiet \
       "gcc_impl_${build_platform}" \
       "gxx_impl_${build_platform}" \
       "gfortran_impl_${build_platform}" \
-      "binutils_impl_${target_platform}=${binutils_version}" \
       "gcc_impl_${target_platform}" \
       "gxx_impl_${target_platform}" \
       "gfortran_impl_${target_platform}" \
       "${c_stdlib}_${target_platform}=${c_stdlib_version}" \
-      "binutils_impl_${cross_target_platform}=${binutils_version}" \
-      "${cross_target_stdlib}_${cross_target_platform}=${cross_target_stdlib_version}" \
       ${extra_pkgs[@]}
+
+    if [[ "${cross_target_cxx_stdlib}" == "libcxx" ]]; then
+      CONDA_OVERRIDE_OSX=15.5 CONDA_SUBDIR="${cross_target_platform}" conda create -p $SRC_DIR/cf-compilers-target -c conda-forge/label/sysroot-with-crypt -c conda-forge --use-local --yes --quiet libcxx-devel
+      mkdir -p ${CF_PREFIX}/${TARGET}/lib
+      ln -sf $SRC_DIR/cf-compilers-target/lib/libc++* ${CF_PREFIX}/${TARGET}/lib
```

Comment:
It the symlink target here intentionally still using the (apparently now obsolete) pattern of
```
$PREFIX/<target>/lib
```
?

Reply:
Yes, it's needed for now until we have built the compilers without `$PREFIX/<target>/lib` once.